Framework/Algorithms/CMakeLists.txt

```diff
@@ -360,6 +361,7 @@ set ( INC_FILES
 	inc/MantidAlgorithms/CalculateDIFC.h
 	inc/MantidAlgorithms/CalculateEfficiency.h
 	inc/MantidAlgorithms/CalculateFlatBackground.h
+    inc/MantidAlgorithms/CalculateIqt.h
```

Comment:
The added lines in the CMakeLists.txt file are not inline with everything else. Ensure that if tabs/spaces are used, it is consistent for the new lines.